tests/rancher/rancher_ui.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+# Maintainer: George Gkioulis <ggkioulis@suse.com>
+
+use base 'x11test';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use x11utils 'ensure_unlocked_desktop';
+
+sub run {
+    my ($self) = shift;
+
+    # switch to desktop
+    if (!check_var('DESKTOP', 'textmode')) {
```

Comment:
If `DESKTOP` is `textmode`, how would the rest of the test work?

Reply:
Fair point, I ll just remove the check since it does not make much sense to include the rest of the test in the `if` block. 
Deciding whether to run this or not will then be left to the scheduling.

Reply:
Fair point, I ll just remove the check since it does not make much sense to include the rest of the test in the `if` block.
Deciding whether to run this or not will then be left to the scheduling.